tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/stamp/RecordStampInbuiltFunctionTest.java

```diff
@@ -539,8 +539,8 @@ public void testStampComplexRecordToJSON() {
         Assert.assertEquals(((BMapType) employee0.getType()).getConstrainedType().getClass(), BAnydataType.class);
 
         Assert.assertEquals((employee0.get("marks").getType().getClass()), BArrayType.class);
-        Assert.assertEquals(((BArrayType) ((BArrayType) (employee0).get("marks").getType()).getElementType()).
-                getElementType().getTag(), TypeTags.INT_TAG);
+        Assert.assertEquals(((BArrayType) (employee0).get("marks").getType()).getElementType().getTag(),
```

Comment:
Not introduced by this PR, but I think the name of the test is misleading? Shall we fix it in a separate PR maybe?

Reply:
I'll send a seperate PR

Reply:
I think we still use "stamp" in our tests. ( even in test class names as well). That needs to change as well.

Reply:
https://github.com/ballerina-platform/ballerina-lang/issues/30326